src/Symfony/Component/Stopwatch/Section.php

```diff
@@ -85,7 +85,7 @@ public function get($id)
     public function open($id)
     {
         if (null === $id || null === $session = $this->get($id)) {
-            $session = $this->children[] = new self(microtime(true) * 1000, $this->morePrecision);
+            $session = $this->children[] = new self(\hrtime(true) / 1e6, $this->morePrecision);
```

Comment:
I am not sure if Symfony uses the leading slash 🧐

Reply:
It seems to on some global functions.

Reply:
As mentioned by fabbot, the blackslash should be removed. We only add them when PHP is able to optimize the call.

Reply:
Does PHP have a hard coded list of functions to optimize, or is it able to do all functions that are defined before runtime?

Reply:
I guess the fact that microtime is not with a slash answers my own question...

Reply:
Here is the full list https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/37cc9c62f71a4cd49d9427aea729954f0ca35fe2/src/Fixer/FunctionNotation/NativeFunctionInvocationFixer.php#L346